internal/main.go

```diff
@@ -145,6 +146,8 @@ func fetchMetadata(provider string) (providers.Metadata, error) {
 		return packet.FetchMetadata()
 	case "openstack-metadata":
 		return openstackMetadata.FetchMetadata()
+	case "cloudstack":
```

Comment:
Let's use `--cloudstack-configdrive` and name the package cloudStackConfigDrive. We'll be adding support for the network metadata service eventually. We can adopt a similar naming convention for OpenStack.